src/main/java/com/backend/webproject/controller/AdminSideController.java

```diff
@@ -0,0 +1,209 @@
+package com.backend.webproject.controller;
+
+import com.backend.webproject.jtemp.Coupons;
+import com.backend.webproject.jtemp.Events;
+import com.backend.webproject.jtemp.JdbcTemplateCoupons;
+import com.backend.webproject.jtemp.JdbcTemplateEvents;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.util.Date;
+import java.util.List;
+
+import javax.servlet.http.HttpServletRequest;
+
+@Controller
+@RequestMapping("/admin-side")
+public class AdminSideController {
+
+    @Autowired
+    JdbcTemplateCoupons jdbcTemplateCoupons;
+
+    @Autowired
+    JdbcTemplateEvents jdbcTemplateEvents;
+
+    @RequestMapping("/")
+    public String showAdminPage() {
+        return "admin";
+    }
+
+    // Shows all couponsjose
+    @RequestMapping("/coupons")
+    public String showCouponsPage(Model model) {
+        try {
+            List<Coupons> newCoupons = jdbcTemplateCoupons.getNewCoupons();
+            model.addAttribute("newCoupons", newCoupons);
+            return "coupons";
+        } catch (Exception e) {
+            System.out.println("Couldn't reach your request");
+        }
+        return "coupons";
```

Comment:
You have redundant returns in all your endpoints. You can erase the ones inside the trys.